src/include/sof/audio/coefficients/src/src_tiny_int16_2_3_1814_5000.h

```diff
@@ -14,14 +14,14 @@ const int16_t src_int16_2_3_1814_5000_fir[48] = {
 	-37,
 	-520,
 	-150,
-	1392,
+	1391,
```

Comment:
Any idea on how the rounding mode was changed in here?

Reply:
These files are generated with Octave using [src_tiny_int16.m](https://github.com/thesofproject/sof/blob/main/tools/tune/src/src_tiny_int16.m) and instructions from [here](https://thesofproject.github.io/latest/algos/src/sample_rate_conversion.html).

Reply:
I'd assume a different build of Octave or something. Shouldn't actually affect sound quality perceptibly so it's fine.

Reply:
Yep, I'd not be worried about those.